geany.nsi.in

```diff
@@ -216,6 +221,17 @@ Section "GTK 2.24 Runtime Environment" SEC06
 	File /r "gtk\lib\*"
 	SetOutPath "$INSTDIR\share\themes"
 	File /r "gtk\share\themes\*"
+	SetOutPath "$INSTDIR\share\icons"
+	File /r "gtk\share\icons\*"
+	SetOutPath "$INSTDIR\share\glib-2.0"
+	File /r "gtk\share\glib-2.0\*"
+!if ${GTK_VERSION} > 2
+	SetOutPath "$INSTDIR\share\gtk-3.0"
+	File /r "gtk\share\gtk-3.0\*"
+!else
+	SetOutPath "$INSTDIR\share\gtk-2.0"
+	File /r "gtk\share\gtk-2.0\*"
+!endif
```

Comment:
gtk/share/icons, glib-2.0 and gtk-2.0 doesn't exist in the GTK bundle (not in the stripped one we distribute).
So icons and glib-2.0 should be moved into the GTK3 branch and the gtk-2.0 directory can be removed from the list.